sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,201 @@
+
+
+## The Material Commons
+
+So if the central quality of materiality here is the ability for something to be remade into something new without having been intended as such, what happens to the original? Channelling Heidegger for hiphop, Charles Mudede understands this transformation (something into its unintended) as arising from breakage, which brings to light the inner, unrecognized potential of an object. The broken object, now marvellously _enstranged_, has been knocked "out of the slumber of its primary function... it is now wide-awake, alert, alive" ("The Turntable"). To make its meta-music, the turntable must, in some way, fail to function as intended. Twigger Holroyd's knitted transformations are irreversible--the transition from jumper to cardigan is permanent, invariably involving some form of creative destruction to enact its renewal. But turntables and sweaters are objects with a very high degree of materiality, indeed. The digital is surely somehow different. Adapting Mudede's poetics, my artistic software aims at becoming hypnagogic media: alive at the threshold between sleep and consciousness, able to move in either direction, to wake up into something new or fall back into the same dream without shattering in transition<sup>14</sup>. Right now, my own pieces are slumbering on Github<sup>15</sup>, waiting to be awoken as the vector for a new work of art by me or someone else.
+
+Whatever happens to the turntable and the analogue vinyl record, James Brown, after all, is never broken by a digital sampler's manipulation of him, nor is his (or the sampler's) essence somehow lost or activated by the process. Transformation is general and ubiquitous when it is digital. Although it may seem that it is digital media's reproducibility that affords such zero cost transformation without breakage, reproducibility, in itself, is just another form of breakdown. Every software programmer knows the impossible tangle of duplication that is the result of the copy/paste procedure. So perhaps it is not the digital's infinite reproducibility that allows it to avoid destruction in the act of recreation, but its capture within a logic of signals and loops: the repetition, modulation, and filtration of materials into something new. "It is this act, replaying, that marks the real rupture in the mode or method of production" (Mudede).
```

Comment:
Comment from David: "I love this claim the claim that reproducibility, in itself, is just another form of breakdown, but I think it needs to be explicated a bit. I also wonder what the word ‘just’ discloses (intentionally, or not), especially so fast on the heels of a Heidegger mention (wherein the word ‘mere’ is unpacked)."